sktime/forecasting/base/adapters/_tbats.py

```diff
@@ -89,21 +91,23 @@ def _fit(self, y, X=None, fh=None):
         """
         self._forecaster = self._instantiate_model()
         self._forecaster = self._forecaster.fit(y)
+        self._yname = y.name
 
         return self
 
-    def _predict(self, fh, X=None, return_pred_int=False, alpha=DEFAULT_ALPHA):
+    def _predict(self, fh, X, return_pred_int=False, alpha=DEFAULT_ALPHA):
         """Forecast time series at future horizon.
 
         Parameters
         ----------
         fh : int, list, np.array or ForecastingHorizon
             Forecasting horizon
-        X : pd.DataFrame, optional (default=None)
-            Exogenous time series
+        X : (default=None)
+            NOT USED BY TBATS
         return_pred_int : bool, optional (default=False)
             If True, returns prediction intervals for given alpha values.
-        alpha : float or list, optional (default=0.95)
+        alpha : float, optional (default=0.05)
+            Interpreted as "Confidence Interval" = 1 - alpha
```

Comment:
unclear what you mean. As coverage = 1-alpha, or upper/lower quantiles (mirrored)?

Reply:
good enough for now